### PR TITLE
Learning

### PR DESCRIPTION
**Pull request by IhorOzerov**

### CODE REVIEW

**Comment by artem-trubin** [comment]
Hey! Please check "Files changed" tab and make sure only your personal changes are there. It seems like your autoformatter is turned on and it messed up the file.

**Comment by artem-trubin** [comment]
Okay, now you are not "adding" your greeting, you are "replacing" the whole file with it. Please make sure you are "adding" a greeting.

File: README.md [hunk 1]
```diff
@@ -1,2857 +1,4 @@
-[![Kottans-Frontend](https://img.shields.io/badge/%3D%28%5E.%5E%29%3D-frontend-yellow.svg)](https://github.com/kottans/frontend)
-[![Kottans-Backend](https://img.shields.io/badge/%3D%28%5E.%5E%29%3D-backend-blue.svg)](https://github.com/kottans/backend)
-[![Kottans-Git](https://img.shields.io/badge/%3D%28%5E.%5E%29%3D-git-orange.svg)](https://github.com/kottans/git-course)
-
 Hi! It's Ihor and I learn GitHub, at least i am tryyy
```
Comment (IhorOzerov) [inline README.md:1]: !

**Comment by IhorOzerov** [comment]
Check out please

**Comment by artem-trubin** [comment]
You can check it yourself, look at "Files changed" tab.

**Comment by IhorOzerov** [comment]
Now i see my PR in "Files changed" tab, but doesn't exist in kottans repo. Is there new problems or thats ok?

**Comment by artem-trubin** [comment]
"Files changed" tab show you lines of code that will be added or removed in the result of this PR's merge. You goal is to make sure that only your changes are in this tab. Right now you are making a lot of changes, it is not normal. 
image

You can fix this by doing something with your branch, it will affect this PR accordingly. If you struggle with "Fixing" those issues, you can view git history on your branch with `git log`, find the last commit that was fine (before any of your changes) and use `git reset` on that commit. After that *carefully* add your new changes. You might have to push those changes with `-f` as `git reset` removes commits from timeline (removes changes completely).

**Comment by IhorOzerov** [comment]
https://github.com/kottans/mock-repo/pull/810/commits
I'm created new branch, deleted previous one. In file "Files changed" see only my PR and in this tab only my changes. 
That's right?


30 июля 2022, 15:17:51, от "Artem Trubin" ***@***.***>:


"Files changed" tab show you lines of code that will be added or removed in the result of this PR's merge. You goal is to make sure that only your changes are in this tab. Right now you are making a lot of changes, it is not normal.
You can fix this by doing something with your branch, it will affect this PR accordingly. If you struggle with "Fixing" those issues, you can view git history on your branch with git log, find the last commit that was fine (before any of your changes) and use git reset on that commit. After that carefully add your new changes. You might have to push those changes with -f as git reset removes commits from timeline (removes changes completely).

**Comment by artem-trubin** [comment]
I'll leave comments in your new PR.

**Comment by IhorOzerov** [comment]
I'm already edited file, check it please

29 июля 2022, 17:26:54, от "Artem Trubin" ***@***.***>:


Hey! Please check "Files changed" tab and make sure only your personal changes are there. It seems like your autoformatter is turned on and it messed up the file.

**Comment by artem-trubin** [comment]
What exactly do you want me to check? Was it a mistake comment? I believe your task here is done long time ago.

**Pull request closed without merging**